src/azure-cli/azure/cli/command_modules/role/_params.py

```diff
@@ -73,23 +73,23 @@ def load_arguments(self, _):
                         'Specifies the URLs where user tokens are sent for sign-in, or the redirect URIs '
                         'where OAuth 2.0 authorization codes and access tokens are sent.')
 
-        # keyCredentials
-        c.argument('start_date', arg_group='keyCredentials',
+        # keyCredential
+        c.argument('start_date', arg_group='keyCredential',
```

Comment:
The property is name `keyCredentials` but the resource type is [`keyCredential`](https://docs.microsoft.com/en-us/graph/api/resources/keycredential?view=graph-rest-1.0), so we use `keyCredential` for simplicity.